griptape/templates/rules/json_schema.j2

```diff
@@ -1 +1,2 @@
-You must respond with a JSON object that successfully validates against the following schema: {{json_schema}}
+You must respond with a JSON object that successfully validates against the following Schema. Do not output markdown or any other text.
+Schema: {{ json_schema }}
```

Comment:
The prompt I've been using that seems to work pretty well is:

```
Output valid JSON that matches this schema: {schema}
No markdown, code snippets, code blocks, or backticks.
```

Reply:
I like that, updated!